.github/workflows/jira_create_and_push_staging_testing_deploy_tags.yml

```
@@ -0,0 +1,120 @@
+name: 'Create and push tag from package version'
+
+on:
+  workflow_dispatch:
+    inputs:
+      commitHash:
+        description: 'Hash du commit à déployer (ex: cadd172232c80206107e2f0122542adf19fb42af)'
+        required: true
+        type: string
+
+permissions:
+  contents: read
+  id-token: write
+
+jobs:
+  create_and_push_staging_tag:
+    runs-on: [self-hosted, linux, x64, regular]
+    env:
+      GIT_CONFIG_EMAIL: github-actions-bot@passculture.app
+      GIT_CONFIG_NAME: ${{ github.actor }}
+    steps:
+      - name: Checkout commit hash
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.commitHash }}
+          token: ${{ secrets.PUSH_SERVICE_ACCOUNT }}
+      - name: get package version
+        id: package-version
+        uses: martinbeentjes/npm-get-version-action@v1.3.1
+      - name: Author
+        run: |
+          git config --global user.email "$GIT_CONFIG_EMAIL"
+          git config --global user.name "$GIT_CONFIG_NAME"
+      - name: Create and push tag from package version
+        run: |
+          VERSION=${{ steps.package-version.outputs.current-version}}
+          TAG_NAME=v${VERSION}
+          git tag -a "$TAG_NAME" -m "🚀 $TAG_NAME"
```

pour rendre plus explicite et maximiser la lisibilité, je recommande de toujours utiliser les options longues (lorsqu'il y en a) dans les scripts et pipeline

```suggestion
          git tag --annotate "$TAG_NAME" --message "🚀 $TAG_NAME"
```

---

il y a d'autres commandes dans le meme cas, je ne spam pas en commentant chacune